File: python/nivel 5 simulacion/28.py
"""
Ejercicio 28
Calcular promedio de notas y determinar aprobación
"""
cantidad = int(input("¿Cuantas notas vas a ingresar? : "))

suma = 0

for i in range(cantidad):
  nota = float(input(f"Ingresa la nota {i + 1} : "))
  suma += nota

promedio = suma / cantidad

print("Promedio: ", round(promedio, 2))

if promedio >= 4.0:
  print("Resultado: APROBADO")
else:
  print("Resultado: REPROBADO")

File: javascript/nivel 5 simulacion/28.js
/*
Ejercicio 28
Calcular promedio de notas y determinar aprobación
*/
let cantidad = parseInt(prompt("¿Cuantas notas vas a ingresar?: " ));

let suma = 0;

for(let i = 1; i <= cantidad; i++){
  let nota = parseFloat(prompt(`ingresa el valor la nota ${i}: `));
  suma += nota;
}

let promedio = suma / cantidad;

console.log("Promedio: ", promedio.toFixed(2));

if(promedio >= 4.0){
  console.log("Resultado: Aprobado");
}else{
  console.log("Resultado: Rechazado");
}